webapp/components/searchable_user_list.jsx

```diff
@@ -147,15 +161,41 @@ export default class SearchableUserList extends React.Component {
                     />
                 );
             }
+
+            if (this.props.infinite) {
+                if (this.state.nextDisabled) {
+                    pageNavLinks = (
+                        <div className='more-modal__loading-bar'>
+                            {'Loading...'}
+                        </div>
+                    );
+                } else {
+                    pageNavLinks = (
```

Comment:
We can probably remove or refactor this bit.  The Not Loading was meant to be a spike as I was developing this.